db/migrate/20250627212857_add_last_seen_to_redirections.rb

```diff
@@ -0,0 +1,10 @@
+class AddLastSeenToRedirections < ActiveRecord::Migration[8.0]
+  def up
+    add_column :redirections, :last_seen_at, :datetime
+    Redirection.update_all(last_seen_at: Time.zone.now)
```

Comment:
Hmm, this isn't quite true, and it might give a false sense of when they were last seen. I think it should either be:

* `NULL` (and fall back to `created_at` in the view, as you wrote)
* Or set it to `created_at` for each one, make it `null: false`, and update it on creation

I think I prefer to set it to `NULL`, which is more accurate: we don't really have a value for it.

Reply:
I forgot I added that fallback logic for `NULL`. I'll do your second option.

Reply:
I will say this has a odd side-effect that if it's `nil` it gets sorter before others that have been seen. I'll see if I can fix that.

Reply:
Tried to solve it in 2cadbca Let me know if you can think of a better API though.